dydx-testnet-4/gentx/gentx-Keplr.json

```diff
@@ -0,0 +1 @@
+{"body":{"messages":[{"@type":"/cosmos.staking.v1beta1.MsgCreateValidator","description":{"moniker":"Keplr","identity":"55AE6F8C61BF4749","website":"https://www.keplr.app/","security_contact":"","details":"Interchain infra. Apoapsis to the Periapsis."},"commission":{"rate":"0.000000000000000000","max_rate":"0.200000000000000000","max_change_rate":"0.010000000000000000"},"min_self_delegation":"1","delegator_address":"dydx13ncafgjak39vp03fw3kxjffme7usrp096rqh35","validator_address":"dydxvaloper13ncafgjak39vp03fw3kxjffme7usrp09lakp69","pubkey":{"@type":"/cosmos.crypto.ed25519.PubKey","key":"LzJRNPGgYEqqLgBbcK/IeGnwB6CwC2pvxQr9F2Qfc1I="},"value":{"denom":"adv4tnt","amount":"500000000000000000000000"}}],"memo":"fe11005e26f1834c0690f3c720f121f6c824a868@172.31.14.186:26656","timeout_height":"0","extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[{"public_key":{"@type":"/cosmos.crypto.secp256k1.PubKey","key":"A93pK58kLw5L5sfj3S/OzoPDuQt/nodveN5kTYvaC/P+"},"mode_info":{"single":{"mode":"SIGN_MODE_DIRECT"}},"sequence":"0"}],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""},"tip":null},"signatures":["nPzPes/WjN/C5LB8c2x9zSDYl8anHGAXuxlz80F11gg36JThiFrh2ZMz7c4ldk6bStZRM91D+hpKa1mIJQuK+Q=="]}
```

Comment:
Hi, @daniel-ck89 
can you check your `gentx`?
`failed to execute message; message index: 0: cannot set validator commission to less than minimum rate of 0.050000000000000000: commission cannot be less than min rate
`

Reply:
updated. thanks